go.mod

```diff
@@ -35,9 +35,9 @@ require (
 	github.com/onsi/ginkgo/v2 v2.23.4
 	github.com/onsi/gomega v1.37.0
 	github.com/opencontainers/image-spec v1.1.0
-	github.com/openshift/api v0.0.0-20240517002838-931fa9860c01
-	github.com/openshift/client-go v0.0.0-20231218140158-47f6d749b9d9
-	github.com/openshift/library-go v0.0.0-20240207105404-126b47137408
+	github.com/openshift/api v0.0.0-20250313134101-8a7efbfb5316
+	github.com/openshift/client-go v0.0.0-20241203091221-452dfb8fa071
```

Comment:
I’d tend to pick `github.com/openshift/client-go@release-4.19` rather than the older commit the openshift installer is using?

Reply:
>  we can try to use branch specific but I think it will again use latest commit from branch.

yes, it will use the latest commit from the branch, which is newer than the one you used. The commits used vary depending on the module you look at, `oc` uses a different set of commits, some newer, some older https://github.com/openshift/oc/blob/release-4.19/go.mod#L36-L39
This is why I’d tend to use the latest commits from the 4.19 branch.